src/controllers/event/addOrganizer.ts

```diff
@@ -0,0 +1,57 @@
+import * as Interfaces from "@interfaces";
+import * as Errors from "@errors";
+import * as Utils from "@utils";
+import { prisma } from "@utils/prisma";
+import { EventOrganiser } from "@prisma/client";
+
+const addOrganizer: Interfaces.Controller.Async = async (req, res, next) => {
+  const { eventId: EID } = req.params;
+  const eventId = String(EID);
+
+  if (!eventId || eventId.length !== 24)
+    return next(Errors.Module.invalidInput);
+
+  const { organizers }: { organizers: [string] } = req.body;
+
+  if (!organizers) return next(Errors.Module.invalidInput);
+
+  if (!organizers.every((organizer) => organizer.length === 24)) {
+    return next(Errors.Module.invalidInput);
+  }
+
+  const results = await Promise.all(
+    organizers.map(async (organizer: string) => {
+      const user = await prisma.user.findFirst({ where: { id: organizer } });
+      if (!user) {
+        return false;
+      }
+      return true;
+    })
+  );
+
+  if (!results.every((result) => result)) {
+    return next(Errors.User.userNotFound);
+  } else {
+    const eventOrganisers: EventOrganiser[] = [];
+    await Promise.all(
+      organizers.map(async (id: string) => {
+        const eventOrganizer = await prisma.eventOrganiser.upsert({
+          create: {
+            user: { connect: { id: id } },
+            event: { connect: { id: eventId } },
+          },
+          where: { userId_eventId: { userId: id, eventId: eventId } },
+          update: {},
```

Comment:
If possible remove the `update` field here, it should not change the functionality according to my knowlege but if it is not removable without changing functionality then leave it as it is.

Reply:
bhaiya update can't be removed (it throws error)

Reply:
ok no issues.